docker/ssp-debug-sp/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM webdevops/php-nginx:7.2 AS ssp-debug-sp
```

Comment:
Why the "ancient" 7.2 and not 8.1 ?

Reply:
Good point, and I tested if we could use a more recent version. But the currently installed SSP version restricts us to using a 7.2 PHP.